<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envelopes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Expenses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Expenses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>19</number>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="leExpenseUserInput">
            <property name="placeholderText">
             <string>Enter new expense here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddExpense">
            <property name="text">
             <string>Add expense</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Envelopes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Envelopes</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="leNewEnvelope">
            <property name="placeholderText">
             <string>Enter name for new envelope here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddEnvelope">
            <property name="text">
             <string>New envelope</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>19</number>
          </attribute>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Expenses for selected envelope:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>19</number>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Rules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblRules">
          <property name="text">
           <string>Rules:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="twRules">
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>19</number>
          </attribute>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnApplyRules">
            <property name="text">
             <string>Apply rules</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Financial Plan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lblBusinessPlan">
          <property name="text">
           <string>Business plan:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lblWeeklyStats">
            <property name="text">
             <string>Weekly stats:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="twBusinessPlan">
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>19</number>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="cbItemType"/>
          </item>
          <item>
           <widget class="QLineEdit" name="leNewBPItem"/>
          </item>
          <item>
           <widget class="QComboBox" name="cbItemFrequency"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbAddBPItem">
            <property name="text">
             <string>Add item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbApplyPlan">
            <property name="text">
             <string>Apply plan</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAddExpense</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addExpense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget_2</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectedEnvelopeChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leExpenseUserInput</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>addExpense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddEnvelope</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addEnvelope()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNewEnvelope</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>addEnvelope()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApplyRules</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>applyRules()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAddBPItem</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addBusinessPlanItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNewBPItem</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>addBusinessPlanItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApplyPlan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>applyBusinessPlan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>653</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addExpense()</slot>
  <slot>onSelectedEnvelopeChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
  <slot>addEnvelope()</slot>
  <slot>applyRules()</slot>
  <slot>addBusinessPlanItem()</slot>
  <slot>applyBusinessPlan()</slot>
 </slots>
</ui>
